community/bolt/src/test/java/org/neo4j/bolt/v1/messaging/MessageFormatTest.java

```diff
@@ -73,22 +65,22 @@
     @Test
     public void shouldHandleCommonMessages() throws Throwable
     {
-        assertSerializes( new RunMessage( "CREATE (n) RETURN åäö" ) );
-        assertSerializes( new DiscardAllMessage() );
-        assertSerializes( new PullAllMessage() );
-        assertSerializes( new RecordMessage( record( 1L, "b", 2L ) ) );
-        assertSerializes( new SuccessMessage( new HashMap<>() ) );
-        assertSerializes( new FailureMessage( Status.General.UnknownFailure, "Err" ) );
-        assertSerializes( new IgnoredMessage() );
-        assertSerializes( new ResetMessage() );
-        assertSerializes( new InitMessage( "MyClient/1.0" ) );
+//        assertSerializes( new RunMessage( "CREATE (n) RETURN åäö" ) );
+//        assertSerializes( new DiscardAllMessage() );
+//        assertSerializes( new PullAllMessage() );
+//        assertSerializes( new RecordMessage( record( 1L, "b", 2L ) ) );
+//        assertSerializes( new SuccessMessage( new HashMap<>() ) );
+//        assertSerializes( new FailureMessage( Status.General.UnknownFailure, "Err" ) );
+//        assertSerializes( new IgnoredMessage() );
+//        assertSerializes( new ResetMessage() );
```

Comment:
I see, only InitMessage is a common messages, all others are uncommon or special messages? :)